src/shared/headers/index.ts

```diff
@@ -28,27 +28,22 @@ export function headersWithToken(clientName: string, clientVersion: string, toke
   };
 }
 
-export function headersWithTokenAndMnemonic(clientName: string, clientVersion: string, token: Token, mnemonic: string) {
+export function headersWithTokenAndMnemonic(clientName: string, clientVersion: string, token: Token) {
```

Comment:
If mnemonic is no longer needed, then I think its better to rename this function to just 'headersWithToken'.. so it will be easier to understand in the future